.ci/openshift-ci/cluster/configs/selinux.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+SELINUX=permissive
```

Comment:
Not something to change now, but would you mind to open an issue about running what we have with SELINUX enabled?

Reply:
That deployment will be applied only if `SELINUX_PERMISSIVE=yes` is set (see https://github.com/kata-containers/tests/pull/3139/commits/a23eb649638693d9a492340ea5e9aa29fde01704#diff-0e9f34ff52b582c56c2f0f81b858ceddd1d9ee77ae0c4b21ebb6a532e05c0184R150). Usually I set this variable on my environment just to debug or test the install script itself. I don't plan to change the default selinux configuration on the nodes in the production CI environment. Makes sense?

Reply:
It does, thanks!